core/admin/EE_Admin.core.php

```diff
@@ -569,6 +582,7 @@ public function wp_loaded()
      */
     public function admin_init()
     {
+        $loader = LoaderFactory::getLoader();
```

Comment:
you could instead add a `$loader` property to the class and change this line to:

```php
$this->loader = LoaderFactory::getLoader();
```

and then use `$this->loader` from within `adminInitModelsReady()` instead of having to use the `LoaderFactory` again.

This also has the benefit of making it easier in the future to inject the `Loader` and simply replace the one line of code where the `LoaderFactory` is being used.

Reply:
ya good idea